tests/Umbraco.Tests.Integration/Umbraco.Core/Services/ContentTypeSchemaServiceTests.cs

```diff
@@ -0,0 +1,76 @@
+using NUnit.Framework;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Tests.Common.Testing;
+using Umbraco.Cms.Tests.Integration.Testing;
+
+namespace Umbraco.Cms.Tests.Integration.Umbraco.Core.Services;
+
+[TestFixture]
+[UmbracoTest(Database = UmbracoTestOptions.Database.NewSchemaPerTest)]
+internal sealed class ContentTypeSchemaServiceTests : UmbracoIntegrationTestWithContentEditing
+{
+    // Required to register IPublishedContentTypeCache (even though it uses in-memory cache, not HybridCache)
+    protected override void CustomTestSetup(IUmbracoBuilder builder)
+        => builder.AddUmbracoHybridCache();
+
+    private IContentTypeSchemaService ContentTypeSchemaService =>
+        GetRequiredService<IContentTypeSchemaService>();
+
+    private IContentTypeService ContentTypeService =>
+        GetRequiredService<IContentTypeService>();
+
+    private IMediaTypeService MediaTypeService =>
+        GetRequiredService<IMediaTypeService>();
+
+    [Test]
+    public void Can_Get_DocumentTypes()
+    {
+        // Act
+        var result = ContentTypeSchemaService.GetDocumentTypes();
+
+        // Assert - count should match the content type service
+        Assert.That(result, Has.Count.EqualTo(ContentTypeService.Count()));
+
+        var contentTypeSchema = result.FirstOrDefault(ct => ct.Alias == ContentType.Alias);
+        Assert.That(contentTypeSchema, Is.Not.Null);
+        Assert.Multiple(() =>
+        {
+            Assert.That(contentTypeSchema!.Alias, Is.EqualTo("umbTextpage"));
+            Assert.That(contentTypeSchema.SchemaId, Is.EqualTo("UmbTextpage"));
+            Assert.That(contentTypeSchema.Properties, Has.Count.EqualTo(1));
+        });
+
+        // Verify property details - the base class creates a "title" property
+        var titleProperty = contentTypeSchema!.Properties.First();
+        Assert.Multiple(() =>
+        {
+            Assert.That(titleProperty.Alias, Is.EqualTo("title"));
+            Assert.That(titleProperty.EditorAlias, Is.EqualTo("Umbraco.TextArea"));
+            Assert.That(titleProperty.Inherited, Is.False);
```

Comment:
```suggestion
            Assert.That(titleProperty.Inherited, Is.False);
            Assert.That(titleProperty.DeliveryApiClrType.FullName, Is.EqualTo("System.String"));
```

As this property wasn't being asserted.